app/views/game_search/show.html.erb

```diff
@@ -31,22 +32,42 @@
       <p><b>Platforms:</b> <%@game.platforms.each do |platform| %>
           <%=platform.platform %>
           <%end %></p>
-        
+
       <table>
         <tr>
           <th class="horiz-center"><b>SIMILAR GAMES</b></th>
+          <th class="horiz-center"><b>REVIEWS</b></th>
         </tr>
         <tr>
-          <td> 
+          <td>
           <%@game.similars.each do |similar| %>
           <%= button_to similar.game, "/game_search?similar=#{similar.game}", method: :post, class: "btn transparent-50 width-wide"%>
           <%end %>
           </td>
+          <td class="grid-space">
+            <p>Number of reviews (<%= @game.reviews.count %>)</p>
+            <center><button class="review"><%= link_to 'Write a Review', "/games/#{@game.id}/review" %></button></center>
+            <br>
+            <div class="scrolling-box">
+            <% if @game.reviews.count == 0 %>
```

Comment:
I'm not sure why `@game.reviews.nil?` didn't work so the work around was to set it ==0